packages/mui-material/src/Checkbox/Checkbox.js

```diff
@@ -117,6 +117,7 @@ const Checkbox = React.forwardRef(function Checkbox(inProps, ref) {
     indeterminateIcon: indeterminateIconProp = defaultIndeterminateIcon,
     inputProps,
     size = 'medium',
+    disableRipple = false,
```

Comment:
What's with this change?

Reply:
Otherwise, `variants` won't work because `disabled=undefined`

```js
variants: [
  { props: { color: 'default', disableRipple: false } }, style: { … }
]
```